mmrazor/models/mutables/mutable_channel/units/mutable_channel_unit.py

```diff
@@ -249,6 +268,58 @@ def _register_channel_container(
                     module.register_mutable_attr('out_channels',
                                                  container_class(out_channels))
 
+        """
+        for module in model.modules():
+            # [blocker]
+            if isinstance(module, MultiheadAttention):
```

Comment:
if isinstance(module, MultiheadAttention) -> if isinstance(module, DynamicMultiheadAttention)

Reply:
Maybe this change is better?
```
if isinstance(module, (DynamicChannelMixin, DynamicMHAMixin)):
    ...
```